### PR TITLE
only show none complete milestones in missed milestones

### PR DESCRIPTION
**Pull request by dtaylor7**
- speed up database queries
- only show none complete milestones in missed milestones

### CODE REVIEW

File: models/milestoneFieldEntry.js [hunk 1]
```diff
@@ -33,6 +33,21 @@ class MilestoneFieldEntry extends Model {
...
     });
   }
+
+  static createSearch(milestoneField, options) {
+    const filters = options.map(option => {
+      return {
+        [Op.and]: {
+          milestone_field_id: milestoneField.id,
+          value: {
+            [Op.like]: `%${option}%`
```
Comment (gds-gharper) [inline models/milestoneFieldEntry.js:43]: double ended like filters are incredibly inefficient (and can never hit indexes either).  You also have the problem where substrings in values will match as well.

I'd prefer it if we just did exact matches, any reason not to (other than allowing people to search for substrings)?

Comment (dtaylor7) [reply models/milestoneFieldEntry.js:43]: this was here before to query blobs, but I guess now you have changed to TEXT we can query normally?

Comment (dtaylor7) [reply models/milestoneFieldEntry.js]: have updated

Comment (gds-gharper) [reply models/milestoneFieldEntry.js]: sure, if you can do that quickly then I'm much happier

**Pull request merged**